### DISCUSSION TITLE
AP should not take control of your cursor.

### DISCUSSION THREAD

**Discussion by Redageddon** [ppy/osu#22593] [Ideas]
In the same way that RX does not take control of your buttons, and still allows you to press buttons, AP should not override the player cursor. Instead it should just ignore any cursor based iteration. I read using my mouse and keys at the same time, but sometime I want to practice a map without having to worry about aiming, and having AP take control of my mouse throws me off by a lot.

**Comment by Rekunan** [comment]
I think a better solution would either be having two separate cursors: ingame and mouse, show up while playing AP (ingame is what AP controls and mouse is what you control) or having two ingame cursors, and possibly one of them could be faded to still differentiate control.

**Reply by cdwcgt** [reply]
I think it should the same as ReplayLoaded
